### PR TITLE
Nikita/groot 79 enable reset password at databankgreen

### PR DESCRIPTION
**Pull request by nikita-pardeshi-github**
Hi,

This PR is for following ticket: [groot-79]( https://linear.app/bankgreen/issue/GROOT-79/enable-reset-password-at-databankgreen)

As part of this ticket, user should be able to reset his password without login into the portal. As part of this story, I have overridden default login.html and some more html files. Added respective paths in urls.py file and added smtp configurations in settings.py file(This is added to send a password_reset email). 

Kindly review my PR. Thanks.

### CODE REVIEW

File: bankgreen/settings.py [hunk 1]
```diff
@@ -109,6 +109,13 @@
 
 DATABASES = {"default": {"ENGINE": "django.db.backends.sqlite3", "NAME": BASE_DIR / "db.sqlite3"}}
 
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.office365.com"
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = "alerts@bank.green"
+EMAIL_HOST_PASSWORD = "knwfjbzrjfdwmdmj"
```
Comment (Chris33871) [inline bankgreen/settings.py:117]: Is this password supposed to be public?

**Review by Chris33871 — COMMENTED** [review]
Hi @nikita-pardeshi-github ,
The reset password feature works, both the scenario where the email exists as well as the one where it doesn't work. There are just a couple of things here and there that need to be amended I think.

File: bankgreen/urls.py [hunk 2]
```diff
@@ -53,4 +54,29 @@
...
+        ),
+        name="password_reset",
+    ),
+    path(
+        "password_reset/done/",
+        auth_views.PasswordResetDoneView.as_view(),
+        name="password_reset_done",
+    ),
+    path(
+        r"^reset/<uidb64>/<token>/",
```
Comment (Chris33871) [inline bankgreen/urls.py:71]: I am getting a warning in Django here. I think you're not supposed to start the path with "^"
image

File: bankgreen/settings.py (hunk 1 above)
Comment (nikita-pardeshi-github) [reply bankgreen/settings.py]: Thank you for reviewing it. Good catch. No, I had stored those parameters as env variables. Seems like,it just got misplaced when I did hard reset. Anyways, I have stored them as env variable in .env file now.

File: bankgreen/urls.py (hunk 2 above)
Comment (nikita-pardeshi-github) [reply bankgreen/urls.py]: Updated urls.py file. I don't see warning message now.

**Comment by Chris33871** [comment]
Hi @nikita-pardeshi-github ,

Thank you for the changes. The warning message is gone but the password can still be viewed in the commit history. This is a security issue, you need to delete those previous commits (I think you might be able to use the git squash command). I am not well versed in git commands so it's worth asking @sopoour or @brugo-dev about this as I think they are more familiar with this.

**Review by Chris33871 — APPROVED** [review]
Hi @nikita-pardeshi-github ,

The changes work. Thank you very much.
I would suggest getting a second person to review this too just to make sure, especially since the changes affect a large codebase.

**Review by RogerTangos — APPROVED** [review]
I've gone through this and everything looks good to me. I'm just waiting to test with credentials locally and then will merge and deploy. Please wait for me to do that.

**Comment by RogerTangos** [comment]
Tested and working. I've also reset the password so that it's not in github's PR version either. Will send to the team :)

**Pull request merged**